### DISCUSSION TITLE
unable to detect btcd version

### DISCUSSION THREAD

**Discussion by nguyenHoang32** [btcsuite/btcd#2251] [Q&A]
Getting the block count was successful. Block: 861271.
However, when I call `client.GetBlockChainInfo()`, I get an error: "Unable to detect btcd version. Status code: 403, response: ''".

`package main

import (
	"fmt"
	"log"

	"github.com/btcsuite/btcd/rpcclient"
)

func main() {
	// RPC server details
	username := "1"
	password := "1"

	// Create a new RPC connection configuration
	connCfg := &rpcclient.ConnConfig{
		Host:         "go.getblock.io/66247efe0af84dfab9814fc368fb473a",
		User:         username,
		Pass:         password,
		HTTPPostMode: true,
		DisableTLS:   false, // Set to true if you don't use TLS
	}
	fmt.Println(connCfg.Host)
	// Create a new RPC client
	client, err := rpcclient.New(connCfg, nil)
	if err != nil {
		log.Fatalf("Error creating new RPC client: %v", err)
	}
	defer client.Shutdown()
	block, err := client.GetBlockCount()
	fmt.Println("Block", block)
	if err != nil {
		log.Fatalf("Error retrieving blockchain info: %v", err)
	}
	// Retrieve blockchain info
	blockchainInfo, err := client.GetBlockChainInfo()
	if err != nil {
		log.Fatalf("Error retrieving blockchain info: %v", err)
	}

	// Print blockchain info
	fmt.Printf("Chain: %s\n", blockchainInfo.Chain)

}
`

**Comment by razorbohan** [comment]
Any solution to that issue? I'm receiving the same error using go.getblock.io RPC node.
